server/lib/alert-manager.js

```diff
@@ -104,9 +117,9 @@ ${$t('server.email.signature')}
                 for (let alert of alerts) {
                     if (!alert.testBalance(balance))
                         continue;
-
+                    let currency = account.currency || defaultCurrency;
                     let text =
-                        alert.formatAccountMessage(account.title, balance);
+                        alert.formatAccountMessage(account.title, balance, currency);
```

Comment:
Again, I'd prefer the format function to be passed to formatAccountMessage from the above, so from here. Maybe the format function can be created before this loop, too?